examples/libsql/README.md

```diff
@@ -25,41 +49,105 @@ curl --request POST \
 }'
 ```
 
-## File structure
+Fetch all the users as JSON:
 
-- `src/index.ts` - the main entry point
-- `src/schema.ts` - defines the database schema
-- `migrations` - contains the SQL migrations generated with Drizzle Kit
-- `src/env.ts` - loads, validates and exports the environment variables
-- `src/server.ts` - the server definition
-- `src/utils.ts` - utility functions
+```bash
+curl --url http://localhost:3000/users
+```
 
-## Migrations
+Stop the server with ctrl-C.
 
-- `pnpm generate` - generate a new migration based on schema changes
+## Switch from local SQLite to remote Turso
 
-Migrations are run automatically when the server starts.
+You must have the [Turso CLI] installed and authenticated to create and manage a
+Turso database.
+
+### Set up the Turso database
+
+Create a new database:
+
+```bash
+turso db create drizzle-example
+```
+
+Create tables by copying the SQL from the [migration] into the Turso CLI shell:
```

Comment:
The migrations are run automatically on the server startup, you don't need to run them manually like this

Reply:
Removed step